examples/layout_flowbox_example.py

```diff
@@ -1,5 +1,5 @@
 import gi
-gi.require_version('Gtk', '3.0')
+gi.require_version('Gtk', '3.12')
```

Comment:
No, Flowbox was introduced in 3.12 as you can see on the [docs](https://developer.gnome.org/gtk3/unstable/GtkFlowBox.html): "Since: 3.12"

I wondered why the example does not work on my Ubuntu 14.04 and it's because that has GTK3.10. Having this fail fast in this case will save people hours of tracking down wondering why the example doesn't work.

Reply:
`3.0` refers to the typelibs version number. There has never been any versions that exist other than `2.0`, `3.0`, and `4.0`.

Reply:
I agree with TingPing, this needs to be reverted.